components/centos/centos-mesos-marathon-singlenode-setup/README.rst

```diff
@@ -0,0 +1,50 @@
+Vagrantfile: CentOS Mesos Marathon Single-Node Setup
+====================================================
+
+This Vagrantfile sets up ADB for development with Mesos and Marathon. It also sets up private networking that will be used to expose the services to the host. This Vagrantfile is based on `CentOS 7 <http://cloud.centos.org/centos/7/vagrant/x86_64/images/>`_ vagrant box. It sets up Mesos master & slave and Marathon on the CentOS 7 system. It uses the rpm packages for these from `Mesosphere repo <http://repos.mesosphere.com/el/7/noarch/RPMS/mesosphere-el-repo-7-1.noarch.rpm>`_.
+
+Libvirt
+-------
+
+To start the Vagrant box with libvirt provider on Fedora 22 and above, install the package ``vagrant-libvirt`` using ``dnf`` as follows:
+
+::
+
+$ sudo dnf install vagrant-libvirt
+
+It is assumed here that libvirt is already installed on the system. If not, you can setup Virtualization on the system using:
+
+::
+
+$ sudo dnf groupinstall Virtualization
+
+Start the Vagrant box using:
+
+::
+
+$ sudo vagrant up --provider=libvirt
+
+Virtualbox
+----------
+
+To start the Vagrant box with virtualbox provider, make sure that VirtualBox is installed. To download using ``dnf``, enable the repo using `this file <http://download.virtualbox.org/virtualbox/rpm/fedora/virtualbox.repo>`_. After that, install it using:
+
+::
+
+$ sudo dnf --enablerepo=virtualbox --disablerepo=rpmfusion-free-updates --disablerepo=rpmfusion-free install VirtualBox-4.3.x86_64
+
+We need to disable a few repos as VirtualBox is available from there as well.
+
+After that, start the box using:
```

Comment:
Installation steps for providers are already listed in the documents so no need to put again, instead create a link to refer existing docs (https://github.com/projectatomic/adb-atomic-developer-bundle/blob/master/docs/installing.rst)